extensions/ipynb/src/deserializers.ts

```diff
@@ -53,6 +53,14 @@ const orderOfMimeTypes = [
 	'text/plain'
 ];
 
+function isEmptyVendoredMimeType(outputItem: NotebookCellOutputItem) {
+	if (outputItem.mime.startsWith('application/vnd.')) {
+		try {
+			return Buffer.from(outputItem.data).toString().length === 0;
```

Comment:
Is this the same as checking `outputItem.data.byteLength === 0` or maybe to be safe we can check `outputItem.data.byteLength === 0 || Buffer.from(outputItem.data).toString().length === 0`